docs/_downloads/433d4b2ff389555700f26fe734f79b9a/plot_group_lasso.py

```diff
@@ -0,0 +1,215 @@
+# the N20 response.
+
+###########################################
+# Solve the inverse problems with groupmne
+# ----------------------------------------
+
+# The Group Lasso assumes the source locations are the same across subjects
+# for all instants i.e if a source is zero for one subject, it will be zero
+# for all subjects. "alpha" is a hyperparameter that controls this structured
+# sparsity prior. it must be set as a positive number between 0 and 1.
+# With alpha = 1, all the sources are 0.
+
+stcs = compute_group_inverse(fwds, evokeds, noise_covs,
+                             method='grouplasso',
+                             spatiotemporal=True,
+                             alpha=0.8)
+
+############################################
+# Let's visualize the N20 response. The stimulus was applied on the right
+# hand, thus we only show the left hemisphere. The activation is exactly in
+# the Primary somatosensory cortex. We highlight the borders of the post
+# central gyrus.
+
+
+t = 0.02
+t_idx = stcs[0].time_as_index(t)
+view = "lateral"
+
+for stc, subject in zip(stcs, subjects):
+    g_post_central = mne.read_labels_from_annot(subject, "aparc.a2009s",
+                                                subjects_dir=subjects_dir,
+                                                regexp="G_postcentral-lh")[0]
+    n_sources = [stc.vertices[0].size, stc.vertices[1].size]
+    m = abs(stc.data[:n_sources[0], t_idx]).max()
+    surfer_kwargs = dict(
+        background="white", foreground="black",
+        clim=dict(kind='value', pos_lims=[0., 0.2 * m, m]),
+        hemi='lh', subjects_dir=subjects_dir,
+        initial_time=t * 1e3, time_unit='ms', size=(500, 500),
+        smoothing_steps=5, cortex=("gray", -1, 6, True))
+    brain = stc.plot(**surfer_kwargs, views=view)
+    brain.add_text(0.1, 0.9, subject + "_groupmne", "title")
+    brain.add_label(g_post_central, borders=True, color="green")
+
+#####################################
+# Group MNE leads to better accuracy
+# ----------------------------------
+# To evaluate the effect of the joint inverse solution, we compute the
+# individual solutions using `mne.inverse_sparse.mixed_norm` for each subject.
+# The group solutions are better located in S1.
+
+
+from mne.inverse_sparse import mixed_norm  # noqa: E402
+
+t = 0.02
+t_idx = stcs[0].time_as_index(t)
+view = "lateral"
+for subject, evoked, fwd, cov in zip(subjects, evokeds, fwds, noise_covs):
+    ev = evoked.copy()
+    ev.pick_types(meg="grad")
+    ev.crop(0.015, 0.025)
+    stc = mixed_norm(ev, fwd, cov, alpha=80., loose=0.)
+    stc.subject = subject
+    g_post_central = mne.read_labels_from_annot(subject, "aparc.a2009s",
+                                                subjects_dir=subjects_dir,
+                                                regexp="G_postcentral-lh")[0]
+    n_sources = [stc.vertices[0].size, stc.vertices[1].size]
+    m = abs(stc.data[:n_sources[0], t_idx]).max()
+    surfer_kwargs = dict(
```

Comment:
plot_kwargs?

Reply:
actually you could avoid a lot of code duplication with this kwargs and highlight was kwargs change between plots